app/src/main/res/layout/cl_fragment_edit_profile.xml

```diff
@@ -0,0 +1,309 @@
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/sv_edit_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/cv_edit_image"
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_marginStart="@dimen/spacing_16"
+                android:layout_marginTop="@dimen/spacing_16"
+                android:layout_marginEnd="@dimen/spacing_6"
+                android:layout_marginBottom="@dimen/spacing_6"
+                app:layout_constraintBottom_toTopOf="@+id/cv_edit_company"
+                app:layout_constraintEnd_toStartOf="@+id/cv_edit_lastName"
+                app:layout_constraintStart_toStartOf="@+id/sv_edit_layout"
+                app:layout_constraintTop_toTopOf="@+id/sv_edit_layout">
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/cl_image_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
+
+                    <de.hdodenhof.circleimageview.CircleImageView
+                        android:id="@+id/iv_edit_image"
+                        android:layout_width="@dimen/spacing_62"
+                        android:layout_height="@dimen/spacing_62"
+                        android:layout_marginHorizontal="@dimen/spacing_27"
+                        android:layout_marginTop="@dimen/spacing_53"
+                        android:src="@drawable/ic_default_user"
+                        app:layout_constraintBottom_toTopOf="@+id/tv_edit_image"
+                        app:layout_constraintCircleRadius="@dimen/spacing_10"
+                        app:layout_constraintEnd_toEndOf="@+id/cl_image_layout"
+                        app:layout_constraintStart_toStartOf="@+id/cl_image_layout"
+                        app:layout_constraintTop_toTopOf="@+id/cl_image_layout"/>
+
+                    <TextView
+                        android:id="@+id/tv_edit_image"
+                        style="@style/TextViewCommonStyle.ProfileImage"
+                        android:layout_marginTop="@dimen/spacing_7"
+                        android:layout_marginBottom="@dimen/spacing_53"
+                        android:text="@string/edit"
+                        app:layout_constraintBottom_toBottomOf="@+id/cl_image_layout"
+                        app:layout_constraintEnd_toEndOf="@+id/cl_image_layout"
+                        app:layout_constraintStart_toStartOf="@+id/cl_image_layout"
+                        app:layout_constraintTop_toBottomOf="@+id/iv_edit_image"/>
+                </androidx.constraintlayout.widget.ConstraintLayout>
+            </androidx.cardview.widget.CardView>
+
+            <androidx.cardview.widget.CardView
+                android:id="@+id/cv_firstName"
```

Comment:
In the View ids, need to avoid the Capital letters, (ex. cv_firstName). Should be like **cv_first_name**. Check and update in all places